examples/dogs/db/migrations/20170328203852_create_dogs_table.js

```diff
@@ -1,10 +1,10 @@
 exports.up = function (knex, Promise) {
-  return knex.schema.createTableIfNotExists('Dogs', function (table) {
+  return knex.schema.createTableIfNotExists('dogs', function (table) {

```

Comment:
@pietgeursen any reason for lowercase table names? we rolled with uppercase on 41S i think?